Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Build the manager binary
-FROM docker.io/library/golang:1.19 as builder
+FROM --platform=$BUILDPLATFORM docker.io/library/golang:1.19 AS builder
+
+ARG BUILDPLATFORM
```

Comment:
Can we remove this argument as it's not necessary?

Reply:
Theoretically it is needed in the FROM statement, but podman complains if it is stated before it...
I would leave it here to ensure it needs to be set.

Reply:
BUILDPLATFORM is implicitly available when using docker buildx and doesn't need to be explicitly declared using ARG unless you're using it somewhere else in the build process.

Reply:
I can drop it sure

Reply:
Done